cpp/include/Ice/OutputStream.h

```diff
@@ -406,49 +407,58 @@ namespace Ice
             }
         }
 
-        /**
-         * Writes a list of mandatory data values.
-         */
+        /// Writes a single non-optional value to this output stream.
+        /// @tparam T The type of the value.
+        /// @param v The value to write.
         template<typename T> void writeAll(const T& v) { write(v); }
 
-        /**
-         * Writes a list of mandatory data values.
-         */
+        /// Writes two or more non-optional values to this output stream.
+        /// @tparam T The type of the first value.
+        /// @tparam Te The types of the remaining values.
+        /// @param v The first value to write.
+        /// @param ve The remaining values.
         template<typename T, typename... Te> void writeAll(const T& v, const Te&... ve)
         {
             write(v);
             writeAll(ve...);
         }
 
-        /**
-         * Writes a list of mandatory data values.
-         */
+        /// @cond INTERNAL
```

Comment:
Using `@privatesection / @publicsection` does not work-around this bug. `@cond INTERNAL` does.

Without this work-around, the entire page is unusable.